automation/afterSignHook.ts

```diff
@@ -0,0 +1,13 @@
+import { notarize } from 'electron-notarize'
+const { ELECTRON_SKIP_NOTARIZATION } = process.env
+
+export async function sign (filePath: string): Promise<void> {
+  const appleId = 'accounts+apple@balena.io'
+
+  await notarize({
+    appBundleId: 'io.balena.etcher',
```

Comment:
In `package.json`, there is a `macos.identifier: "io.balena.cli"` value that I believe is currently used by oclif for signing (not notarizing) the installer. I wonder if that value would (eventually) work here.